hack/test

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -eux -o pipefail
```

Comment:
why is this file added? can it be combined with the existing test.sh?

Reply:
You are correct, should have not been committed. Dropping.